### PR TITLE
OKTA-520454 Add group exists operation to SCIM protocol

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Add group exists operation to SCIM protocol
- **Is this PR related to a Monolith release?** No

### Preview:
https://664797f19e6c500cf3b16aff--reverent-murdock-829d24.netlify.app/docs/reference/scim/scim-20/#retrieve-groups

### Resolves:

* [OKTA-520454](https://oktainc.atlassian.net/browse/OKTA-520454)
* [OKTA-518815](https://oktainc.atlassian.net/browse/OKTA-518815)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md](https://github.com/okta/okta-developer-docs/blob/c331529c3125d5686048856e46e43e601bb193ef/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fscim%2Fscim-20%2Findex.md) | 78 | 136 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7c446774-8d3c-406d-a059-637f81ca45a5) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.ecdcd7e0-dddc-4903-9610-e7d38fd203c1?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md](https://github.com/okta/okta-developer-docs/blob/54b2ef3f533eb52ed983bd952445f270e9320832/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fscim%2Fscim-11%2Findex.md) | 77 | 140 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/45175e92-ae8e-4fa5-9b59-27d470a6b411) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md](https://github.com/okta/okta-developer-docs/blob/54b2ef3f533eb52ed983bd952445f270e9320832/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fscim%2Fscim-20%2Findex.md) | 78 | 136 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f901253f-0687-44b3-b9fd-412b620120f0) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 2 of 2 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.f8ef5d80-2cd2-4fa9-9940-0433653accc0?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md](https://github.com/okta/okta-developer-docs/blob/0499c00d519efaf458ca41c6c1ee2a74dfc69bc7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fscim%2Fscim-11%2Findex.md) | [77](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0cbd14fc-5742-4b24-9f34-698bf660aa19) | 82  | 84  | 59  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md](https://github.com/okta/okta-developer-docs/blob/0499c00d519efaf458ca41c6c1ee2a74dfc69bc7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fscim%2Fscim-20%2Findex.md) | [78](https://okta.acrolinx.cloud/api/v1/checking/scorecards/16229b9e-d611-4e36-848f-2a3b02142878) | 82  | 86  | 59  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.657e50cb-2d15-4bb5-9129-46d762d93d7a?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md](https://github.com/okta/okta-developer-docs/blob/4786d0cf4ee1052212336d794bebc6368dc6ea5b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fscim%2Fscim-11%2Findex.md) | [76](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d26510a4-4f62-45b6-a68d-7f2521c2cec0) | 82  | 82  | 59  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md](https://github.com/okta/okta-developer-docs/blob/4786d0cf4ee1052212336d794bebc6368dc6ea5b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fscim%2Fscim-20%2Findex.md) | [78](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5e713021-b0c3-4a4a-b9a8-aa59750775c8) | 82  | 84  | 60  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.b46551a2-512c-4fd7-888b-35e52ac32131?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md](https://github.com/okta/okta-developer-docs/blob/c432af97f1648757d32c66025d8b3a8e7bd2e90b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fscim%2Fscim-11%2Findex.md) | [76](https://okta.acrolinx.cloud/api/v1/checking/scorecards/87d2d620-d880-4611-9313-8805e6ae4f32) | 82  | 82  | 59  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md](https://github.com/okta/okta-developer-docs/blob/c432af97f1648757d32c66025d8b3a8e7bd2e90b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fscim%2Fscim-20%2Findex.md) | [78](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2a160d34-aafb-4c1d-a119-0554c7f2f106) | 82  | 84  | 60  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.bd1ccbfc-9412-426e-886c-ba2e194b102b?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md](https://github.com/okta/okta-developer-docs/blob/ae8bd02b9b5126eb7ad906d90a5a2910a1d2f685/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fscim%2Fscim-11%2Findex.md) | [76](https://okta.acrolinx.cloud/api/v1/checking/scorecards/cfb0d7fa-c584-4b16-b0db-eaf9ce67fda5) | 82  | 82  | 59  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md](https://github.com/okta/okta-developer-docs/blob/ae8bd02b9b5126eb7ad906d90a5a2910a1d2f685/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fscim%2Fscim-20%2Findex.md) | [78](https://okta.acrolinx.cloud/api/v1/checking/scorecards/fb017efb-a32b-42ea-9300-889e30fcbc04) | 82  | 84  | 60  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.dfc69877-4461-4731-b9a4-ff4ad9b78cc8?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md](https://github.com/okta/okta-developer-docs/blob/cdc088071b7ddf0482bf9d33a08f3d3ba47dac42/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fscim%2Fscim-11%2Findex.md) | [76](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e1f88ddf-c6c9-447c-bbd4-a6204f5e3911) | 81  | 80  | 62  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md](https://github.com/okta/okta-developer-docs/blob/cdc088071b7ddf0482bf9d33a08f3d3ba47dac42/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fscim%2Fscim-20%2Findex.md) | [78](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c4fd8189-ed18-4c87-b7cc-6fbcbb18feda) | 82  | 82  | 62  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.572ac260-b795-4c51-9abc-083dd60e06db?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md](https://github.com/okta/okta-developer-docs/blob/9bc5f4340716223714534c3e2b59c1003e91312a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fscim%2Fscim-11%2Findex.md) | [76](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e3f1ebca-1d05-4b42-bd6f-ebae6ceb9590) | 81  | 81  | 62  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md](https://github.com/okta/okta-developer-docs/blob/9bc5f4340716223714534c3e2b59c1003e91312a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fscim%2Fscim-20%2Findex.md) | [78](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8d39fca5-25e7-4b8f-9814-d13e8b7e864d) | 82  | 84  | 62  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.dc4f88c0-5370-42b1-bdba-86c682a99261?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by susanharper-okta — DISMISSED** [review]
LGTM

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md](https://github.com/okta/okta-developer-docs/blob/b4fe0131d59cf8fb9d21cbe9bab5b4ac63319fa4/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fscim%2Fscim-11%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ef1a95d0-647a-436d-b459-67a07d2d877e) | 100  | 87  | 64  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md](https://github.com/okta/okta-developer-docs/blob/b4fe0131d59cf8fb9d21cbe9bab5b4ac63319fa4/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fscim%2Fscim-20%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0dd51a54-b170-4b9e-a4d6-32a0e9cb1c54) | 100  | 90  | 65  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.957b554a-4f28-416f-a031-aa3b82ba2d6f?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md [hunk 1]
```diff
@@ -23,11 +23,11 @@ To better understand SCIM and the specific implementation of SCIM using Okta, se
...
 The User creation operation brings the user's application profile from Okta over to the Service Provider as a User object. A user's application profile represents the key-value attributes defined on the **Profile** tab when a User object is added.
 
-To enable user provisioning, you must configure the provisioning options in the Okta Admin Console. In the Okta Admin Console:
+To enable user provisioning, you must configure the provisioning options in the Admin Console. In the Admin Console:
 
 1. Select your SCIM integration from the list of integrations in your Okta org.
-1. Under the **Provisioning** tab, click **To App** and **Edit**.
-1. In the **Create User** option, click **Enable** and then **Save**.
+1. Click **To App** and **Edit** under the **Provisioning** tab.
+1. Click **Enable** and then **Save** in the **Create User** option.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md:30]: are these options or sections?
```suggestion
1. Click **Enable** and then **Save** in the **Create User** section.
```

File: packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md [hunk 2]
```diff
@@ -342,11 +342,11 @@ Content-Type: text/json;charset=UTF-8
...
 Updating a User object refers to modifying an attribute in the Okta user's application profile that is mapped to an attribute in the SCIM application.
 
-To update a User object, you need to enable the functionality in the Okta Admin Console:
+To update a User object, you need to enable the functionality in the Admin Console:
 
 1. Select your SCIM integration from the list of integrations in your Okta org.
-1. Under the **Provisioning** tab, click **To App**.
-1. In the **Update User Attributes** option, click **Enable** and then **Save**.
+1. Click **To App**, under the **Provisioning** tab.
+1. Click **Enable** and then **Save**, in the **Update User Attributes** option.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md:349]:
```suggestion
1. Click **Enable** and then **Save** in the **Update User Attributes** section.
```

File: packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md [hunk 3]
```diff
@@ -484,13 +484,13 @@ All other updates to User objects are handled through a PUT method request.
...
+The `active` attribute in an Okta user profile represents the user's current status.
 
-To deactivate users, you need to enable the functionality in the Okta Admin Console:
+To deactivate users, you need to enable the functionality in the Admin Console:
 
 1. Select your SCIM integration from the list of integrations in your Okta org.
-1. Under the **Provisioning** tab, click **To App** and **Edit**.
-1. In the **Deactivate Users** option, click **Enable** and then **Save**.
+1. Click **To App** and **Edit** under the **Provisioning** tab.
+1. Click **Enable** and then **Save** in the **Deactivate Users** option.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md:493]:
```suggestion
1. Click **Enable** and then **Save** in the **Deactivate Users** section.
```

File: packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md [hunk 4]
```diff
@@ -539,7 +539,7 @@ Content-Type: text/json;charset=UTF-8
 }
 ```
 
-> **Note:** The SCIM server response to PATCH method requests can also be a HTTP 204 response, with no body returned.
+> **Note:** The SCIM server response to PATCH method requests can also be an HTTP 204 response, with no body returned.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md:542]:
```suggestion
> **Note:** The SCIM server response to PATCH method requests can also be an HTTP 204 response with no body returned.
```

File: packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md [hunk 5]
```diff
@@ -567,7 +567,7 @@ To create a Group object on the SCIM server, you first need to enable provisioni
 1. Select your SCIM integration from the list of integrations in your Okta org.
 2. On the **Push Groups** tab, click **Push Groups**.
 
-You can select which existing Okta group to push, either by specifying a name or a rule. If a group doesn't exist, create a new group in Okta and then push it to the SCIM server. For more information, see [About Group push](https://help.okta.com/okta_help.htm?id=ext_Directory_Using_Group_Push) in the Okta Help Center.
+You can select which existing Okta group to push, either by specifying a name or a rule. If a group doesn't exist, create a group in Okta and then push it to the SCIM server. For more information, see [About Group push](https://help.okta.com/okta_help.htm?id=ext_Directory_Using_Group_Push) in the Okta Help Center.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md:570]:
```suggestion
You can select which existing Okta group to push, either by specifying a name or a rule. If a group doesn't exist, create a group in Okta and then push it to the SCIM server. See [About Group push](https://help.okta.com/okta_help.htm?id=ext_Directory_Using_Group_Push) in the Okta Help Center.
```

File: packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md [hunk 6]
```diff
@@ -33,19 +33,19 @@ To better understand SCIM and the specific implementation of SCIM using Okta, se
 
 The User creation operation brings the user's application profile from Okta over to the Service Provider as a User object. A user's application profile represents the key-value attributes defined on the **Profile** tab when a User object is added.
 
-To enable user provisioning, you must configure the provisioning options in the Okta Admin Console. In the Okta Admin Console:
+To enable user provisioning, you must configure the provisioning options in the Admin Console. In the Admin Console:
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md:36]:
```suggestion
To enable user provisioning, you must configure the provisioning options in the Admin Console:
```

File: packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md [hunk 7]
```diff
@@ -33,19 +33,19 @@ To better understand SCIM and the specific implementation of SCIM using Okta, se
...
 
 1. Select your SCIM integration from the list of integrations in your Okta org.
-1. Under the **Provisioning** tab, click **To App** and **Edit**.
-1. In the **Create User** option, click **Enable** and then **Save**.
+1. Click **To App** and **Edit** under the **Provisioning** tab.
+1. Click **Enable** and then **Save** in the **Create User** option.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md:40]:
```suggestion
1. Click **Enable** and then **Save** in the **Create User** section.
```

File: packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md [hunk 8]
```diff
@@ -457,13 +457,13 @@ All other updates to User objects are handled through a PUT method request.
...
+The `active` attribute in an Okta user profile represents the user's current status.
 
-To deactivate users, you need to enable the functionality in the Okta Admin Console:
+To deactivate users, you need to enable the functionality in the Admin Console:
 
 1. Select your SCIM integration from the list of integrations in your Okta org.
-1. Under the **Provisioning** tab, click **To App** and **Edit**.
-1. In the **Deactivate Users** option, click **Enable** and then **Save**.
+1. Click **To App** and **Edit** under the **Provisioning** tab.
+1. Click **Enable** and then **Save** in the **Deactivate Users** option.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md:466]:
```suggestion
1. Click **Enable** and then **Save** in the **Deactivate Users** section.
```

**Review by susanharper-okta — COMMENTED** [review]
couple comments

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md](https://github.com/okta/okta-developer-docs/blob/9a8c01832e8aa95f2907406e228223dc9bfe8b29/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fscim%2Fscim-11%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9d50606c-58f3-47ff-9757-609a22dd01f3) | 100  | 90  | 64  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md](https://github.com/okta/okta-developer-docs/blob/9a8c01832e8aa95f2907406e228223dc9bfe8b29/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fscim%2Fscim-20%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e60a41be-6ae2-4298-9ea6-23bccda03dbd) | 100  | 90  | 65  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.93480a9f-3231-4634-b6e8-3bf2b7bffc90?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**